src/Arcus.Observability.Correlation/ICorrelationInfoAccessor.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Arcus.Observability.Correlation 
+{
+    /// <summary>
+    /// Represents a contract to access the <see cref="CorrelationInfo"/> model.
+    /// </summary>
+    public interface ICorrelationInfoAccessor : ICorrelationInfoAccessor<CorrelationInfo>
```

Comment:
Why do we have this interface? Just for convenience

Reply:
Yes convenience and for the applications that dont need a special correla info model.